test/models/episode_test.rb

```diff
@@ -493,4 +493,48 @@
       refute_nil episode.ready_transcript
     end
   end
+
+  describe "#increment_asset_wait" do
```

Comment:
Hrm - why is this test on the `Episode`?  Shouldn't this be in one of the Apple modules?